rollup.config.ts

```diff
@@ -367,7 +369,7 @@ function cjs({
       forceDevEnv ? forceEnvPlugin('development') : undefined,
       replace({
         // TODO: figure out a better way to produce extensionless cjs imports
-        "require('./logger.js')": "require('./logger')",
+        // "require('./logger.js')": "require('./logger')",
```

Comment:
feel free to remove those instead of commenting them out as those files no longer exist.